psi4/driver/driver_cbs.py

```diff
@@ -1415,8 +1415,11 @@ def cbs(func, label, **kwargs):
     core.set_variable('CBS NUMBER', Njobs)
 
     # new skeleton wavefunction w/mol, highest-SCF basis (just to choose one), & not energy
-    basis = core.BasisSet.build(molecule, "ORBITAL", 'sto-3g')
-    wfn = core.Wavefunction(molecule, basis)
+    basis,ecpbasis = core.BasisSet.build(molecule, "ORBITAL", 'sto-3g')
+    if ecpbasis:
+        wfn = core.Wavefunction(molecule, basis, ecpbasis)
+    else:
+        wfn = core.Wavefunction(molecule, basis)
```

Comment:
These lines should probably be replaced by the modified `core.Wavefunction.build` function.